Compile-and-Flash-STMBoard-Guide/README.adoc

```diff
@@ -0,0 +1,52 @@
+== *Compiling and Flashing STM Board*
+
+----
+This has been tested on Linux Mint and is expected to work on other Debian based distros
+----
+
+All commands will look like `this`
+
+=== Setup
+. Go to UCSolarCarTeam repo >> Recruit Resources, and clone
```

Comment:
`>>` has a special meaning in programming, don't use it for file direction. -> should suffice, or just >.